python/cugraph/dask/pagerank/pagerank.py

```diff
@@ -278,3 +282,32 @@ def get_chunksize(input_path):
         size = [os.path.getsize(_file) for _file in input_files]
         chunksize = max(size)
     return chunksize
+
+
+def _read_csv(input_files, delimiter='\t', names=['src', 'dst'],
+              dtype=['int32', 'int32']):
+    df = []
+    for f in input_files:
+        df.append(cudf.read_csv(f, delimiter=delimiter, names=names,
+                                dtype=dtype))
+    df_concatenated = cudf.concat(df)
+    return df_concatenated
+
+
+def read_split_csv(input_files):
```

Comment:
Can we add a short doc at the beginning since it is exposed? It would be useful for users to know about the use case and limitations.

Reply:
added